src/main/resources/net/gleske/scmfilter/impl/trait/Messages.properties

```diff
@@ -26,3 +26,4 @@ WildcardSCMHeadFilterTrait.DisplayName=Filter by name including PRs destined for
 RegexSCMHeadFilterTrait.DisplayName=Filter by name including PRs destined for this branch (with regular expression)
 WildcardSCMOriginFilterTrait.DisplayName=Filter by name including PRs originating from this branch (with wildcards)
 RegexSCMOriginFilterTrait.DisplayName=Filter by name including PRs originating from this branch (with regular expression)
+WildcardSCMBranchPRFilter.DisplayName=Filter by branch name and PRs destination
```

Comment:
How does this differ from the following?

```
WildcardSCMHeadFilterTrait.DisplayName=Filter by name including PRs destined for this branch (with wildcards)
```

The plugin already does what your description suggests.